src/Spectre.Angular2Client/src/app/plotly/plotly.component.spec.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * preparation.component.spec.ts
+ * Tests of component wrapping Plotly library.
+ *
+   Copyright 2017 Grzegorz Mrukwa
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { PlotlyComponent } from './plotly.component';
+
+describe('PlotlyComponent', () => {
+  let component: PlotlyComponent;
+  let fixture: ComponentFixture<PlotlyComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ PlotlyComponent ]
+    })
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(PlotlyComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
```

Comment:
If this component is a base, which will just be used in `heatmap` component, etc., an additional single test for *some* data should be prepared.